cores/esp8266/Schedule.cpp

```diff
@@ -49,6 +47,7 @@ struct recurrent_fn_t
 
 static recurrent_fn_t* rFirst = nullptr;
 static recurrent_fn_t* rLast = nullptr;
+static uint32_t rTarget;
```

Comment:
maybe a comment stating that `rTarget` will be initialized before the first call to `schedule_function()`

Reply:
I have added a comment about the purpose of this object, C++ rules are clear on that it is zero initialized by default.